tutorials/tts/Audio_Codec_Training.ipynb

```diff
@@ -130,26 +133,29 @@
         "*   **audio_codec_*.yaml**: Audio codec configurations optimized for various sampling rates.\n",
         "*   **mel_codec_*.yaml**: A mel-spectrogram based codec designed to maximize the performance of speech synthesis models.\n",
         "*   **encodec_*.yaml**: A reproduction of the original [EnCodec](https://arxiv.org/abs/2210.13438) model setup.\n",
+        "*   **low_frame-rate_speech_codec_22050.yaml**: The Low Frame-rate Speech Codec: an audio codec that achieves high-quality audio compression with a 1.89 kbps bitrate and 21.5 frames per second.\n",
         "\n",
         "This tutorial can be run with any of our predefined configs. As a default we have selected `audio_codec_16000.yaml`, which works for 16kHz audio."
-      ],
-      "metadata": {
-        "id": "ODgdGgsAAUku"
-      }
+      ]
     },
     {
       "cell_type": "code",
-      "source": [
-        "from omegaconf import OmegaConf"
-      ],
+      "execution_count": null,
       "metadata": {
         "id": "SPtjS2LkzE9Q"
       },
-      "execution_count": null,
-      "outputs": []
+      "outputs": [],
+      "source": [
+        "from omegaconf import OmegaConf"
+      ]
     },
     {
       "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "iCPJFKg63Dsv"
+      },
```

Comment:
I think it is possible for me to edit it using a code editor that does not support notebook editing and then edit directly the metadata. But then, if I run to test it, it will automatically update the metadata. Then I'm not sure how to do so.